match2/commons/match_group_display.lua

```diff
@@ -5,17 +5,78 @@ function MatchGroupDisplay.bracket(frame)
 	return MatchGroupDisplay.luaBracket(frame, args)
 end
 
-function MatchGroupDisplay.luaBracket(frame, args)
+function MatchGroupDisplay.customBracket(frame, args, matches)
+	if not args then
+		args = require('Module:Arguments').getArgs(frame)
+	end
+	if not matches or #matches == 0 then
+		args[1] = args.id or args[1] or ''
+
+		local MatchGroupUtil = require('Module:MatchGroup/Util')
+		matches = MatchGroupUtil.fetchMatches(args[1])
+		if #matches == 0 then
+			error('No Data found for bracketId=' .. args[1])
+		end
+	end
+
+	if (args.title or '') ~= '' then
```

Comment:
```suggestion
	if args.title ~= '' then
```

Or am I missing something?

Reply:
I was. I get why you're doing this, although I do think it would be easier to explicitly compare `nil` and then `''`. Not a must to change these though.